src/extension.ts

```diff
@@ -1097,6 +1098,24 @@ async function getLogsForPod(pod: PodSummary, follow?: boolean) {
     }
 }
 
+async function getEvents(follow: kuberesources.EventDisplayMode, explorerNode?: explorer.ResourceNode) {
```

Comment:
flip the variables, explorerNode should come first since that's the namespace being operated on, with the enum second.

Reply:
the reason for that was we can allow it to be invoked from command palette as well where  we can make it to show events of current Namespace. with that logic behind the optional explorerNode cant be first argument